backend/dataall/modules/mlstudio/api/input_types.py

```diff
@@ -12,17 +13,17 @@
     ],
 )
 
-ModifySagemakerStudioUserProfileInput = gql.InputType(
-    name='ModifySagemakerStudioUserProfileInput',
+ModifySagemakerStudioUserInput = gql.InputType(
+    name='ModifySagemakerStudioUserInput',
     arguments=[
         gql.Argument('label', gql.String),
         gql.Argument('tags', gql.ArrayType(gql.String)),
         gql.Argument('description', gql.String),
     ],
 )
 
-SagemakerStudioUserProfileFilter = gql.InputType(
-    name='SagemakerStudioUserProfileFilter',
+SagemakerStudioUserFilter = gql.InputType(
+    name='SagemakerStudioUserFilter',
```

Comment:
Some test are still have reference to the old name. It should be rename as well.